src/main/java/org/jvnet/hudson/plugins/nextbuildnumber/NextBuildNumberAction.java

```diff
@@ -78,8 +78,8 @@ private void checkPermission(Job job) {
             MultiBranchProject parentProject = (MultiBranchProject) job.getParent();
             try {
                 parentProject.getACL().checkPermission(getPermission());
-            } catch (org.acegisecurity.AccessDeniedException e) {
-                if (parentProject.getParent() instanceof ComputedFolder) {
+            } catch (Exception e) {
```

Comment:
Perhaps we could just switch to [hasPermission](https://javadoc.jenkins-ci.org/hudson/security/ACL.html#hasPermission-hudson.security.Permission-)?

Reply:
I like the multicatch way but I'm having troubles with spring security dependency.

```
[ERROR] updateShouldFailIfJobDoesNotExist(org.jvnet.hudson.plugins.nextbuildnumber.CommandTest)  Time elapsed: 0.28 s  <<< ERROR!
java.lang.NoSuchMethodError: org.springframework.beans.factory.support.RootBeanDefinition.<init>(Ljava/lang/Class;Z)V
	at hudson.util.spring.DefaultBeanConfiguration.createBeanDefinition(DefaultBeanConfiguration.java:201)
	at hudson.util.spring.DefaultBeanConfiguration.getBeanDefinition(DefaultBeanConfiguration.java:180)
	at hudson.util.spring.DefaultBeanConfiguration.addProperty(DefaultBeanConfiguration.java:217)
	at hudson.util.spring.BeanBuilder.setProperty(BeanBuilder.java:551)
```

I'll look into it more tomorrow, for now I'm off.